Optimize 3.1.0 alpha2 (#167)
camunda/blog.camunda.org: content/post/2020/05/camunda-optimize-31-alpha2-released.md

Flow Node and User Task Duration are a great indication for underperformance or bottlenecks within your end-to-end process.
Optimize already allows you to look at minimum, maximum and average durations of Flow Nodes and User Tasks for a long time, but with this release we add filtering capabilities for Flow Node and User Task Durations for the first time.

With the help of this new filter you can simply identify process instances where a certain flow node is taking longer or shorter than you expected.

You can find the new *Flow Node Duration Filter* in the Report Edit mode alongside the Process Instance Duration Filter under the category *Duration*.

{{< figure src="flownode-duration-filter1.png" alt="Flow Node Duration Filter 1" >}}

Within the modal you are able to specify a duration for one or multiple flow nodes at the same time (combined with OR).

{{< figure src="flownode-duration-filter2.png" alt="Flow Node Duration Filter 2" >}}

In the report edit mode you are able to see the applied filters with the help of a small overlay:

{{< figure src="flownode-duration-filter3.png" alt="Flow Node Duration Filter 3" >}}

## Assignee and Candidate Group Filter

> When monitoring performance and execution of User Tasks assignees and candidate groups are an important ingredient.

Reviewer: Reads better with a comma:
```suggestion
When monitoring performance and execution of User Tasks, assignees and candidate groups are an important ingredient.
```

In order to make the analysis and monitoring more efficient we added Assignee and Candidate Group filters within this release.

You will find the new *Assignee and Candidate Group Filters* within the Report Builder Filter menu:

{{< figure src="usertask-assignee-filter1.png" alt="User Task Assignee & Candidate Group 1" >}}

Within the modal you can define which assignees (or candidate groups) should be included in the set of process instances you are looking at:
{{< figure src="usertask-assignee-filter2.png" alt="User Task Assignee & Candidate Group 2" >}}

{{< figure src="usertask-assignee-filter3.png" alt="User Task Assignee & Candidate Group 3" >}}

# New User Task Reports in regards to Assignee & Candidate Groups

In Optimize 3.0 we added User Task Trend Analysis by introducing "group by start & end date" for User Tasks.

> With this release we go even further and allow you to distribute the information even further to look into Assignees or Candidate Groups.

Reviewer: Would be nice to remove the double "even further", maybe like:
```suggestion
With this release we go even further and allow you to distribute the information by assignees or candidate groups.
```
